data/preset/escola.yaml

```diff
@@ -17,22 +17,33 @@ defines:
       shortdef: 'Layout.Box(orientation: vertical)'
       slots:
         contents:
-        - shortdef: Layout.Box
+        - type: Layout.Overlay
           styles:
             - landing
           slots:
-            contents:
-            - shortdef: 'Banner.Dynamic(show-subtitle: true, mode: text, expand: true)'
-        - shortdef: 'ContentGroup.ContentGroup(expand: true)'
-          slots:
-            <<: *main-arrangement-slot
-            selection: Selection.AllSets
-        - shortdef: 'Layout.Box(orientation: vertical, halign: fill, hexpand: true)'
-          styles:
-            - footer
+            content:
+              shortdef: Layout.Box
+              styles:
+                - background
+            overlays:
+            - shortdef: 'Layout.Box(orientation: vertical, valign: center, vexpand: true)'
+              slots:
+                contents:
+                - shortdef: 'Banner.Dynamic(mode: text, expand: true)'
+                - shortdef: 'ContentGroup.StaticText(label: "TV ESCOLA")'
```

Comment:
To clarify what I meant earlier - 'escola' was supposed to be the generic template that we could expose in the portal and allow other people to use. That was the whole reason we forked it from capacitate (since that had the logos in there).

So for our release by the end of march we either need 'another fork' or to have worked out how to parametarise that stuff.